lib/site_prism/page.rb

```diff
@@ -43,14 +45,14 @@ def page
     # Runs load validations on the page, unless input is a string
     #
     # When calling #load, all the validations that are set will be ran in order
-    def load(expansion_or_html = {}, &block)
+    def load(html = nil, with_validations: true, **expansion, &block)
```

Comment:
can we just add the `with_validations` key here so that it's not backwards incompatible please. As this is quite a widely used method.

Reply:
`def load(expansion_or_html = {}, with_validations: true, &block)` doesn't work since when you pass in `username: 'foobar'` it blows up with `#<ArgumentError: unknown keyword: username>`.

Another way would be to do a `expansion_or_html.fetch(:with_validations, false)` (before `load_html_website`)

The current change is 100% backwards compatible (assuming specs test every possible use case)

Reply:
The 2nd option you mentioned, that just assumes that no-one nestles that option inside the hash. I think this is a valid assumption (If you don't want to place it as a config option). Because that expansion_or_html hash is only ever realistically going to have a couple of options max in it.